.yamllint_custom.yml

```diff
@@ -0,0 +1,17 @@
+# SPDX-License-Identifier: MIT
+---
+extends: .yamllint.yml
```

Comment:
IMHO it makes more sense to have a file like `.yamllint_defaults.yml` that contains the defaults and a `.yamllint.yml` that extends it. This makes sure that yamllint without special configuration picks up the custom configuration instead of only the defaults. This would be useful for running yamllint inside of IDEs/editors.

Reply:
This is a good point.

Reply:
So the presence of `_custom` in the name of the file doesn't lead you to believe that this is the file that should have local customizations?

Reply:
It makes me understand that it is meant for local customomizations but the `yamllint` tool does not know this. It will only load `.yamllint.yml` by default. Therefore it would be better to have the customizations in `.yamllint.yml` so that `yamllint` can be called by IDEs without arguments specific to system roles.

Reply:
ah - so it is only for IDE support where the IDE cannot pass command line arguments to the `yamllint` command

Reply:
yes, or just to avoid having to configure the IDE specifically for each project, for example for one-time contributors.